doc/interfaces/socketcand.rst

```diff
@@ -37,6 +37,33 @@ The output may look like this::
     Timestamp: 1637791111.609763    ID: 0000031d    X Rx                DLC:  8    16 27 d8 3d fe d8 31 24
     Timestamp: 1637791111.634630    ID: 00000587    X Rx                DLC:  8    4e 06 85 23 6f 81 2b 65
 
+
+This interface also supports :meth:`can.BusABC._detect_available_configs`.
```

Comment:
Refer to `` :meth:`~can.detect_available_configs` `` instead of `BusABC._detect_available_configs`.
Avoid using the private method here.